single-obra.php

```diff
@@ -0,0 +1,106 @@
+<?php
+get_header();
+?>
+
+<!-- main content -->
+
+<main id="main-content" class="container">
+
+  <div class="row">
+
+    <div class="col col-6">
+
+      <ul>
+      <?php
+        // this needs active state for current year. but wp_get_archives doesnt do that so perhaps we add a class with js string matching!?!?
+        $args = array(
+          'type'            => 'yearly',
+          'limit'           => '',
+          'format'          => 'html',
+          'before'          => '',
+          'after'           => '',
+          'show_post_count' => false,
+          'echo'            => 1,
+          'order'           => 'DESC'
+        );
+        wp_get_archives($args);
+      ?>
+        <li>&nbsp;</li>
+        <li><a href="<?php echo home_url('obra/'); ?>"><?php echo __('[:es]Todas[:en]All'); ?></a></li>
+      </ul>
+
+    </div>
+
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+    $meta = get_post_meta($post->ID);
+?>
+
+      <article <?php post_class('u-float'); ?> id="single-work-<?php the_ID(); ?>">
+
+        <div class="col col-9">
+
+          <h2 id="single-work-title" class="font-italic">
+            <?php
+              the_title();
+              if (!empty($meta['_igv_year'])) {
+                echo '<br/>' . $meta['_igv_year'][0];
+              }
+            ?>
+          </h2>
+
+          <div id="single-work-meta">
+            <?php
+              if (!empty($meta['_igv_medium_es']) && !empty($meta['_igv_medium'])) {
+                echo '<div>' . __('[:es]' . $meta['_igv_medium_es'][0] . '[:en]' . $meta['_igv_medium'][0]) . '</div>';
```

Comment:
if we are going to be doing the logic of which field to echo depending on the lang, we should follow some pattern. 
i think `qtranxf_getLanguage()` would return the lang code, we could use this code for postfix the meta id's
that way and have a simpler logic and simpler outputs